amp_conf/htdocs/admin/libraries/modulefunctions.class.php

```diff
@@ -2964,7 +2964,7 @@ function url_get_contents($url,$request,$verb='get',$params=array()) {
 		if(!$amp_conf['MODULEADMINWGET']) {
 			$pest = FreePBX::Curl()->pest($url);
 			try{
-				$contents = $pest->$verb($url.$request,$params);
+				$contents = $pest->$verb($url.$request);
```

Comment:
We send params in here for a reason and now you are just dropping them.

This is now a regression.